apps/backend/src/application/dto/update-application.dto.ts

```diff
@@ -0,0 +1,5 @@
+import { ApplicationStatus } from '@prisma/client';
+
+export class UpdateApplicationDto {
+  applicationStatus: ApplicationStatus;
+}
```

Comment:
either validate the enum here (might have to create a new enum for that, not sure if class-validator can handle the prisma enum), or handle the db error if invalid enum was sent

Reply:
It was fine with the prisma enums